types/vue-apollo.d.ts

```diff
@@ -21,7 +21,7 @@ type VueApolloOptions = {
 export class VueApollo implements PluginObject<{}> {
   [key: string]: any;
   install: PluginFunction<{}>;
-  constructor (options: { defaultClient: ApolloClient<{}>, defaultOptions?: VueApolloOptions });
+  constructor (options: { defaultClient: ApolloClient<{}>, defaultOptions?: VueApolloOptions, clients: { [key: string]:any } });
```

Comment:
@azulite 
thanks! this is my mistake 🙇 
Client property is optional value, so could you change clients property to type of optional value?
https://www.typescriptlang.org/docs/handbook/functions.html#optional-and-default-parameters

And I'm glad to run `yarn run test:types` on your local machine.(Your changes cause to fail tests...)